distributed/scheduler.py

```diff
@@ -4009,7 +4008,7 @@ async def close(self, comm=None, fast=False, close_workers=False):
         Scheduler.cleanup
         """
         parent: SchedulerState = cast(SchedulerState, self)
-        if self.status in (Status.closing, Status.closed, Status.closing_gracefully):
+        if self.status in (Status.closing, Status.closed):
```

Comment:
closing_gracefully is specific of Worker and Nanny.
There will be some iteration on this status - at the very least in the name - when I'll introduce the new "retiring" status for workers.